src/Apps/Collect/CollectApp.tsx

```diff
@@ -61,7 +61,9 @@ export class CollectApp extends Component<CollectAppProps> {
             alignItems="flex-end"
           >
             <Serif size="8">
-              <h1 className="title">Collect art and design online</h1>
+              <h1 className="title" style={h1Style}>
```

Comment:
Curious, do you know what the purpose of this className is?

Reply:
I don't—thought it might be for SEO purposes? But on further reflection that doesn't make much sense; I doubt Google cares about classes. Maybe could be removed?

Reply:
Yeah classname won't have any affect, the `<h1>` should be enough. You can run a lighthouse audit to raise any SEO red flags.  Also: https://accessibility.iu.edu/creating-content/web-content/titles.html

Reply:
Nice, this is a helpful resource. I'll remove the `className="title"`!